frontend/src/map/RandomMap.js

```diff
@@ -5,11 +5,14 @@ class RandomMap extends GameMap {
     constructor() {
         const width = 12;
         const depth = 12;
-        const height = 5;
+        const height = 6;
         super(width, depth, height);
 
-        for (let x = -width/2; x < width/2; x++) {
-            for (let y = -depth/2; y < depth/2; y++) {
+        this.heightMap = [];
```

Comment:
I guess this is fine for now. I'm thinking we should add something to the `GameMap` interface, which can be used for defining things like entry/exit points and other interesting things on the map. I'll put it on my todo list.

Reply:
Sounds like a plan. At some point, we should try to sit down and whiteboard more of the game flow.

Reply:
Agreed